pkg/integrations/prometheus/promql/metricsSearchHandler.go

```diff
@@ -469,7 +469,7 @@ func ProcessGetMetricTimeSeriesRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 	}
 
 	res := segment.ExecuteMultipleMetricsQuery(hashedMNamesList, metricQueriesList, queryFormulas, timeRange, qid)
-	mQResponse, err := res.GetResultsPromQlForUi(metricQueriesList[0], metricQueriesList[0].PqlQueryType, start, end, metricQueriesList[0].Interval)
+	mQResponse, err := res.FetchPromqlMetrics(metricQueriesList[0], metricQueriesList[0].PqlQueryType, start, end, metricQueriesList[0].Interval)
```

Comment:
keep the orig call and keep the new call commented out. When the UI PR comes out then you can clean up